files/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Get the server architecture
+server_arch=$(uname -m)
+
+# Check the server architecture and invoke the appropriate bash script
```

Comment:
I don't get the need of a second/third script.. we can reduce them to a single one. The only difference is the CLI URL??

Reply:
Fixed